src/core/plugins/disableSchemes.js

```diff
@@ -0,0 +1,10 @@
+// OutSystems change: create a plugin that disables the Schemes selection
+const DisableSchemesPlugin = function () {
```

Comment:
This could be a lambda instead: `() => {`

Reply:
done